docs/GettingStartedGuide.md

```diff
@@ -50,7 +50,7 @@ HERE OLP SDK consists of specific libraries, each with distinct functionality.
 
```

Comment:
Comment to line 49
Articles are missing.
Change to:
"[Versioned Layer Read example](./examples/nodejs-read-versioned-layer.md) shows how to retrieve layer metadata, partition metadata, and partition data from a versioned layer using the OLP SDK for TypeScript."